docs/models/dti.rst

```diff
@@ -4,7 +4,171 @@
+in the 20s–30s, then declining with older age. In a sample of 430 subjects 
+aged 8–85 :cite:p:`westlye2010`, fractional anisotropy plateaus by the early 30s 
+and slowly falls thereafter, while mean and radial diffusivities do the reverse 
+(minimal in young adults, then rising in aging). Interestingly, this large study 
+found no simple "last-in-first-out" pattern although late-maturing frontal tracts 
+often showed pronounced aging changes, all regions eventually exhibited 
+microstructural decline, indicating a widespread but heterogeneous aging effect 
+rather than one specific sequence.
+
+**Regional Patterns in Aging**: Many DTI studies of aging report that anterior white 
+matter tracts (which myelinate last) are more vulnerable to aging. Salat et al. 
+found significantly lower FA in older adults (mean age ~67) compared to young 
+(mean ~24), especially in the frontal lobes and corpus callosum :cite:p:`salat2005`. 
+In contrast, posterior tracts like the splenium of the callosum or occipital white 
+matter showed smaller FA differences. Such findings support that age-related myelin 
+degeneration and fiber loss are often greatest in late-developing, more complex pathways 
+(though subsequent research has refined this view with more nuanced patterns).
+
+**Microstructural Changes with Aging**: DTI metrics suggest that aging involves 
+loss of fiber integrity (lower FA) and increased water mobility 
+(higher MD/apparent diffusion coefficient, ADC) :cite:p:`westlye2010`. 
+In older adults, increased radial diffusivity is commonly observed, consistent 
+with demyelination or degraded myelin packing, while axial diffusivity may also 
+eventually decrease if axonal loss occurs. Longitudinal studies in elderly cohorts 
+(e.g. over 60) have confirmed ongoing within-person FA declines annually. These DTI changes correlate with cognitive slowing and 
+executive function decline in many studies, highlighting DTI's value in tracking 
+brain aging and its cognitive consequences.
+
+DTI Methodological Warnings and Caveats
+=======================================
+
+**Single Tensor Limitations (Crossing Fibers)**: The basic DTI model assumes one 
+dominant fiber orientation per voxel – an assumption often violated in the brain. 
+In regions with crossing, kissing, or branching fibers, the tensor model yields 
+an average that can underestimate anisotropy and obscure fiber directions. For 
+instance, a voxel containing two crossing tracts will show an artificially low 
+FA (appearing "isotropic") even if each tract is highly anisotropic. This issue 
+can lead to misinterpretation of reduced FA: it might reflect complex fiber 
+geometry rather than neural degeneration. Advanced high-angular-resolution methods 
+(e.g., multi-tensor or Q-ball imaging) are recommended when crossing fibers are 
+prevalent, or one should interpret DTI metrics in such regions with caution 
+:cite:p:`wheelerkingshott2009`.
```

Comment:
I'm not familiar with this one off the top of my head. Did it get picked because it is early?